### DISCUSSION TITLE
Migration from 4.1.0 to 4.1.1

### DISCUSSION THREAD

**Discussion by akshansh96** [pmndrs/zustand#1324] [General]
Quick question:
I noticed that in the release notes for v4.1.1 we only have some refactoring and fixes in middleware and devtools.
Currently, we are using zustand version v4.1.0 for our react native app
Do we require end to end rigorous testing post migrating from v4.1.0 to v4.1.1?

**Comment by dai-shi** [comment]
Hi, it depends on your team rules. You might want to check full changelog to make super sure.
